README.md

```diff
@@ -1,5 +1,3 @@
-
-[![Gitter](https://badges.gitter.im/mne-tools/mne-bids.svg)](https://gitter.im/mne-tools/mne-bids)
```

Comment:
is there a discourse badge we could use?

Reply:
I tried using https://user-images.githubusercontent.com/1681963/52239617-e2683480-289c-11e9-922b-5da55472e5b4.png but then wasn't sure it it's really helpful… I can add it again for you to get an impression of what it looks like, hang on a minute…

Reply:
@sappelhoff I tried and tried, and everything turned out not to be looking great. I don't want to spend much more time on this – if you have any good idea, go for it :) As a "compromise", for now I've added a large Discourse logo to the "Get Help" section

Reply:
yes, that looks good - thanks!